decidim-core/spec/lib/form_builder_spec.rb

```diff
@@ -95,6 +95,8 @@ def self.model_name
 
           expect(parsed.css("li.tabs-title a").count).to eq 3
 
+          expect(parsed.css(".editor label[for='resource_short_description_en']").first).to be_nil
```

Comment:
I think this is an expected behavior, and it should have tests. Here I'm checking that tabbed inputs do not have a label, while the non-tabbed inputs (see next test) checks that it has one. A tabbed input should not have a label next to it since the label is already added on top of the tab titles. 

See screenshot for what I mean: 

![](https://i.imgur.com/EydiqNQ.png)